CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
     Bug #3997: Almalexia doesn't pace
     Bug #4036: Weird behaviour of AI packages if package target has non-unique ID
     Bug #4047: OpenMW not reporting its version number in MacOS; OpenMW-CS not doing it fully
-    Bug #4125: OpenMW logo cropped on bugtracker
```

Comment:
Why did this need to be removed? From what I understand, it fixed itself by us using GL instead of Redmine?

Reply:
It is not a bug in OpenMW itself at all, it does not have an entry in 0.45 milestone and does not have related code in 0.45 master branch.
So I do not know if the changelog entry is really needed here.
And logo is still cropped on the old bugtracker.

Reply:
I understand, I'm not sure either however... let us get in the habit of focusing only on what the PR is addressing instead of doing unrelated drive-by's. I assume that if it's a problem or irrelevant that Raevol will take care since it's his responsibility. ;) 

Please revert and we'll leave it for Raevol to decide. Good?